### PR TITLE
Fix rwx logo

### PR DESCRIPTION
**Pull request by kantuni**
**rwx** logo is missing. This is the best one I could find - it's from [their GitHub page](https://github.com/rwx-research/). Their current logo is provided as an inline SVG.

Signed-off-by: Henrikh Kantuni <henrikh.kantuni@gmail.com>

### CODE REVIEW

**Comment by Anton-4** [comment]
Hi @kantuni, thanks for your contribution!
I was thinking it could be nicer to store the raw svgs in a sponsor folder and link to those, that should also cause the whole README to load faster. What do you think?

**Comment by thehabbos007** [comment]
> I was thinking it could be nicer to store the raw svgs in a sponsor folder and link to those

Hey, just chiming in since I happen to have a cached version of the README in my browser pointing to the "old" svg :)

<details><summary>rwx SVG</summary>

```svg
<svg
  version="1.1"
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink"
  viewBox="68 44.98 631 248.02"
>
  <title>rwx_banner_transparent</title>
  <g
    id="Banner-Transparent"
    stroke="none"
    stroke-width="1"
    fill="none"
    fill-rule="evenodd"
  >
    <rect id="Rectangle" x="0" y="0" width="766" height="339" />
    <g
      id="rwx_logo"
      transform="translate(348.000000, 111.000000)"
      fill="#008754"
      fill-rule="nonzero"
    >
      <g
        id="Group"
        transform="translate(175.500000, 57.500000) scale(-1, 1) rotate(-180.000000) translate(-175.500000, -57.500000) translate(0.000000, 0.000000)"
      >
        <path
          d="M57.9948522,114.428966 C48.7273331,112.682848 42.0244105,108.899591 34.0391896,100.92565 L28.0939887,94.988847 L27.744271,100.92565 C27.5111259,104.126867 27.2196945,108.201144 27.0448356,109.889058 L26.6951179,112.915664 L13.347559,112.915664 L0,112.915664 L0,56.4578318 L0,0 L14.2801395,0 L28.560279,0 L28.560279,33.8164951 L28.560279,67.6329903 L30.3671538,71.0670234 C33.9809034,77.8186816 38.3523746,81.4855305 46.1627366,84.2793201 C50.1262039,85.6762149 51.1170707,85.7926228 60.9091664,85.7926228 C68.428097,85.7926228 72.9744271,85.501603 76.8796081,84.8031556 C79.9104949,84.2211161 82.4750913,83.8718924 82.5916639,83.9883003 C82.7082365,84.1047082 83.6991033,90.4489388 84.864829,98.0736563 C86.4968449,109.015999 86.7882763,111.9844 86.2054135,112.391828 C85.7974095,112.624644 83.232813,113.264887 80.4933577,113.788723 C73.5572899,115.06921 63.0657589,115.418434 57.9948522,114.428966 Z"
          id="Path"
        />
        <path
          d="M101.709565,111.053137 C101.884424,110.005466 106.48904,84.6285438 111.851378,54.7117133 L121.585188,0.291019752 L134.349884,0.291019752 L147.11458,0.291019752 L154.16722,35.3297979 C158.014115,54.5953054 161.336433,70.4849839 161.511292,70.6595957 C161.686151,70.8342076 164.36732,58.4949701 167.514779,43.1873312 C170.720525,27.9378962 173.984557,12.0482177 174.858851,7.8575333 L176.374294,0.291019752 L189.13899,0.291019752 L201.903686,0.291019752 L211.987214,56.1668121 C217.466124,86.8984979 222.012454,112.217216 222.070741,112.508236 C222.070741,112.741052 216.067253,112.915664 208.781468,112.915664 L195.492195,112.915664 L191.528728,81.3691226 C186.690966,42.4888837 185.816672,36.0282453 185.525241,36.2610611 C185.408668,36.377469 182.319495,52.9655948 178.647459,73.1041616 C174.975423,93.2427284 171.769678,110.412894 171.594819,111.344157 L171.245101,112.915664 L161.569578,112.799256 L151.835769,112.624644 L144.841415,74.2100367 C139.595649,45.2826734 137.788774,36.2610611 137.439057,37.541548 C137.147625,38.5310151 134.991033,55.2937528 132.601295,74.7920762 C130.153271,94.3486035 128.054965,110.878525 127.880106,111.635177 L127.646961,112.915664 L114.474261,112.915664 L101.359847,112.915664 L101.709565,111.053137 Z"
          id="Path"
        />
        <path
          d="M238.565759,112.508236 C238.740618,112.217216 247.133843,99.8779788 257.275656,85.0359714 C267.417469,70.1357602 275.635835,57.6801148 275.460976,57.2726871 C275.286118,56.8652595 267.88376,45.8647129 258.907672,32.768824 C249.98987,19.7311392 241.305214,6.98447404 239.673198,4.48170418 L236.584025,0 L252.263036,0 L267.88376,0 L280.706742,18.3342444 C287.701096,28.4035278 293.646297,36.6684887 293.879442,36.6684887 C294.112587,36.6684887 300.057788,28.4035278 307.168715,18.3342444 L320.049983,0 L335.554135,0 L351,0 L350.009133,1.57150666 C349.42627,2.50276986 340.625042,15.4240468 330.366656,30.3824621 L311.831617,57.5637069 L326.461475,78.9827606 C334.563268,90.7399586 343.073065,103.195604 345.462803,106.687841 L349.775988,112.915664 L334.155264,112.915664 L318.53454,112.915664 L306.29442,95.4544786 C299.591498,85.8508267 293.937728,78.0514974 293.762869,77.9932935 C293.588011,77.9932935 287.992527,85.8508267 281.289605,95.4544786 L269.224344,112.915664 L253.778479,112.915664 C245.268682,112.915664 238.449186,112.741052 238.565759,112.508236 Z"
          id="Path"
        />
      </g>
    </g>
    <g
      id="Group"
      transform="translate(68.000000, 45.000000)"
      fill="#008754"
      fill-rule="nonzero"
    >
      <path
        d="M103.180071,1.81413045 C107.40629,-0.604710151 112.593843,-0.604710151 116.820062,1.81413045 L140.882548,15.591205 C145.272443,17.9683857 148.032427,22.5419986 148.094009,27.541434 C148.155591,32.5408694 145.509106,37.1813411 141.179102,39.6663559 C136.849099,42.1513707 131.515279,42.0908435 127.242556,39.5082065 L110.000067,29.6438211 L92.7575769,39.5082065 C88.4848541,42.0908435 83.1510344,42.1513707 78.8210306,39.6663559 C74.4910269,37.1813411 71.8445415,32.5408694 71.9061239,27.541434 C71.9677063,22.5419986 74.7276896,17.9683857 79.1175852,15.591205 L103.180071,1.81413045 L103.180071,1.81413045 Z M49.7476029,48.270426 C53.5119004,54.8752141 51.2223015,63.2869458 44.632606,67.0623557 L41.4701079,68.8809295 L44.632606,70.6995034 C49.0225017,73.0766841 51.7824849,77.6502969 51.8440673,82.6497324 C51.9056497,87.6491678 49.2591643,92.2896394 44.9291606,94.7746542 C40.5991568,97.259669 35.2653371,97.1991418 30.9926143,94.6165049 L27.5001164,92.6188291 L27.5001164,96.4350787 C27.5001164,104.043947 21.3440354,110.212153 13.7501247,110.212153 C6.15621399,110.212153 6.62274327e-05,104.043947 6.62274327e-05,96.4350787 L6.62274327e-05,68.8809295 C-0.0155450245,63.838855 2.7298023,59.1943114 7.1501287,56.7846581 L30.9926143,43.1453542 C37.5844188,39.3736424 45.9796147,41.667751 49.7476029,48.270426 L49.7476029,48.270426 Z M170.25253,48.270426 C174.020518,41.667751 182.415714,39.3736424 189.007519,43.1453542 L212.836254,56.7846581 C217.248963,59.2040981 219.994973,63.8408459 220,68.8809295 L220,96.4350787 C220,104.043947 213.843919,110.212153 206.250008,110.212153 C198.656098,110.212153 192.500017,104.043947 192.500017,96.4350787 L192.500017,92.6188291 L189.007519,94.6165049 C184.734796,97.1991418 179.400976,97.259669 175.070972,94.7746542 C170.740969,92.2896394 168.094483,87.6491678 168.156066,82.6497324 C168.217648,77.6502969 170.977631,73.0766841 175.367527,70.6995034 L178.530025,68.8809295 L175.367527,67.0623557 C168.777832,63.2869458 166.488233,54.8752141 170.25253,48.270426 L170.25253,48.270426 Z M74.0025883,103.378724 C77.7705765,96.7760494 86.1657725,94.4819407 92.7575769,98.2536526 L110.000067,108.118038 L127.242556,98.2536526 C131.515279,95.6710156 136.849099,95.6104884 141.179102,98.0955032 C145.509106,100.580518 148.155591,105.22099 148.094009,110.220425 C148.032427,115.219861 145.272443,119.793473 140.882548,122.170654 L123.750058,131.979931 L123.750058,151.543377 C123.750058,159.152245 117.593977,165.320452 110.000067,165.320452 C102.406156,165.320452 96.2500748,159.152245 96.2500748,151.543377 L96.2500748,131.979931 L79.1175852,122.170654 C72.5278897,118.395244 70.2382909,109.983512 74.0025883,103.378724 Z M13.7501247,137.766302 C21.3440354,137.766302 27.5001164,143.934509 27.5001164,151.543377 L27.5001164,171.106823 L44.632606,180.9161 C49.0225017,183.293281 51.7824849,187.866894 51.8440673,192.866329 C51.9056497,197.865764 49.2591643,202.506236 44.9291606,204.991251 C40.5991568,207.476266 35.2653371,207.415739 30.9926143,204.833102 L6.93012883,191.056027 C2.64627684,188.604201 0.00172431745,184.040723 0.000133021286,179.097526 L0.000133021286,151.543377 C0.000133021286,143.934509 6.15621399,137.766302 13.7501247,137.766302 Z M206.250008,137.766302 C213.843919,137.766302 220,143.934509 220,151.543377 L220,179.097526 C219.998409,184.040723 217.353856,188.604201 213.070004,191.056027 L189.007519,204.833102 C184.734796,207.415739 179.400976,207.476266 175.070972,204.991251 C170.740969,202.506236 168.094483,197.865764 168.156066,192.866329 C168.217648,187.866894 170.977631,183.293281 175.367527,180.9161 L192.500017,171.106823 L192.500017,151.543377 C192.500017,143.934509 198.656098,137.766302 206.250008,137.766302 Z M74.0025883,213.595321 C77.7705765,206.992646 86.1657725,204.698537 92.7575769,208.470249 L96.2500748,210.467925 L96.2500748,206.651675 C96.2500748,199.042807 102.406156,192.874601 110.000067,192.874601 C117.593977,192.874601 123.750058,199.042807 123.750058,206.651675 L123.750058,210.467925 L127.242556,208.470249 C131.515279,205.887612 136.849099,205.827085 141.179102,208.3121 C145.509106,210.797115 148.155591,215.437586 148.094009,220.437022 C148.032427,225.436457 145.272443,230.01007 140.882548,232.387251 L117.026312,246.054109 C112.700009,248.64863 107.300124,248.64863 102.973821,246.054109 L79.1175852,232.387251 C72.5278897,228.611841 70.2382909,220.200109 74.0025883,213.595321 L74.0025883,213.595321 Z"
        id="Shape"
      />
    </g>
  </g>
</svg>
```

</details>

**Comment by rtfeldman** [comment]
Actually can we put the svg in https://github.com/roc-lang/design-assets instead?

If we're not linking to the sponsor's domain, I'd prefer to link to a URL we control (e.g. I can't tell from the domain who controls `https://avatars.githubusercontent.com/u/97913715` and might change it in the future)

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ We are very grateful to our sponsors [NoRedInk](https://www.noredink.com/), [rwx
 
 [<img src="https://www.noredink.com/assets/logo-red-black-f6989d7567cf90b349409137595e99c52d036d755b4403d25528e0fd83a3b084.svg" height="60" alt="NoRedInk logo"/>](https://www.noredink.com/)
 &nbsp;&nbsp;&nbsp;&nbsp;
-[<img src="https://www.rwx.com/build/_assets/rwx_banner_transparent_cropped-RYV7W2KL.svg" height="60" alt="rwx logo"/>](https://www.rwx.com)
+[<img src="https://avatars.githubusercontent.com/u/97913715" height="60" alt="rwx logo"/>](https://www.rwx.com)
```
Comment (dan-manges) [inline README.md:19]:
```suggestion
[<img src="https://www.rwx.com/rwx_banner.svg" height="60" alt="rwx logo"/>](https://www.rwx.com)
```

**Review by dan-manges — COMMENTED** [review]
If you're good with linking to a sponsor's domain, here's a version that's decoupled from our front-end build process so that the path won't change. We also added a test to our build pipeline to make sure this remains a permanent link.

**Review by Anton-4 — APPROVED** [review]

**Pull request merged**